devices.js

```diff
@@ -12483,6 +12483,13 @@ const devices = [
     },
 ];
 
-module.exports = devices.map((device) =>
-    device.extend ? Object.assign({}, device.extend, device) : device,
-);
+
+module.exports = devices.map((device) => {
+    if (device.extend) {
+        device = Object.assign({}, device.extend, device);
+    }
+
+    device.toZigbee.push(tz.scene_store, tz.scene_recall, tz.scene_add);
```

Comment:
Do we need to add this to all devices? There are quiet a few sensors that have no toZigbee functions as they are read only so scenes for not make sense for them.

Reply:
You are right, will only be added now if a device has at least 1 toZigbee converter. This will still lead to some devices have the scenes converters where they don't support it however it won't cause problems (we do the same for groups).